src/converter.ts

```diff
@@ -300,14 +305,9 @@ export class Converter {
       page += 1
 
       const slide = pptx.addSlide()
-
-      slide.addImage({
+      slide.background = {
         data: `data:image/png;base64,${imageFile.buffer!.toString('base64')}`,
-      })
-
-      const [img] = slide['relsMedia']
-      slide['bkgdImgRid'] = img.rId
-      slide['data'] = []
```

Comment:
We don't have to hack internal objects for supporting background image per slide in PptxGenJS v3.3.0 🙌